hooks/pre_gen_project.py

```diff
@@ -28,4 +32,13 @@
         {{ cookiecutter.update({ '_tailwind_addon': 'yes' }) }}
     {% endif %}
 {% endif %}
+
+-----------
+OpenAPI
+cookiecutter.use_openapi
+-----------
+Only project with web only:
+{% if cookiecutter._web_variant == 'yes' and cookiecutter._both_variant == 'no' %}
```

Comment:
I am unsure of we need `_both_variant`, the same check can be achieved with:

```suggestion
{% if cookiecutter._web_variant == 'yes' and cookiecutter._api_variant == 'no' %}
```